extensions/commands/management/cmd_clean.py

```diff
@@ -0,0 +1,53 @@
+from conan.api.conan_api import ConanAPI
```

Comment:
The name of the "layer" that is "management" probably a bit too generic, but I guess that until we have more commands it is difficult to have a better naming and structure.

Reply:
Yes, I've been trying to find a better name but I'm not being able to find the right word either :/